### ISSUE TITLE
Feature Request - Ability to 'Copy and Rename' an item. Similar to the current option to copy menus.

### ISSUE THREAD

**Issue by dazedwolf** [CoagulateSL/GPHUD#70]
**Is your feature request related to a problem? Please describe.**
A large number of items that we plan on creating will have the same options: "Give, Trade, Use, Discard, Inspect, etc.". A lot of these will be the same script that will pull information from available data. This feature would reduce the amount of manual labor needed in creating each item by already creating a large number of the menu options and interactions already prefilled.

**Describe the solution you'd like**
Just a button when you have the item open that functions just like the copy menu button. It asks you for a new item name and then copies all the settings to it. Description, Weight, Tradeable, Destroyable, and any custom buttons.

**Describe alternatives you've considered**
Making each item one at a time, including every button that is shared on all of them.

**Additional context**
This would be more of a 'nice convenience' than anything. It's not a high-priority feature for us. While having it copy - everything - except the item name would be nice, the core thing we'd need would be to have it copy any of the available actions.

**Comment by dazedwolf** [comment]
Is this feature possible to include within the next major update? While it's not a core feature we're currently waiting on, it would be a nice convenience to have.

**Comment by ronlaws86** [comment]
A way to clone attributes from another item when creating more items would be a nice time saver to save having to manually re-enter the same stuff multiple times in to the admin section of the hud

**Comment by ronlaws86** [comment]
@iprice pleeeasseeee!

**Comment by dazedwolf** [comment]
This feature is no longer needed on our end, we figured out a workaround.
